src/Tizen.Multimedia.Camera/Camera/CameraDeviceManager.cs

```diff
@@ -113,6 +113,36 @@ public IEnumerable<CameraDeviceInformation> SupportedDevices
             }
         }
 
+        /// <summary>
+        /// Gets the product id for given <param name="device"/>.
+        /// </summary>
+        /// <exception cref="ArgumentException">Invalid enumeration.</exception>
+        /// <exception cref="ObjectDisposedException">The CameraDeviceManager already has been disposed.</exception>
+        /// <since_tizen> 11 </since_tizen>
+        public ushort GetProductId(CameraDevice device)
```

Comment:
please consider changing `ushort` -> `UInt16`, those are exactly the same type (one is an alias for another) so nothing needs to change apart form return type, but would be more explicit which is better for public API
same in line 137

Reply:
ushort is exactly same with UInt16. I just want to keep this like other alias types.

Reply:
ok, `ushort` is not used very often, but both styles are used across TizenFX, so I will not object